fake-news-detection/src/fake_news_detection/pipelines/data_processing/nodes.py

```
@@ -0,0 +1,133 @@
+import pandas as pd
+from sklearn.preprocessing import LabelEncoder
```

Ruff (0.11.9)

`sklearn.preprocessing.LabelEncoder` imported but unused

Remove unused import: `sklearn.preprocessing.LabelEncoder`

(F401)